client/dashboard/leaderboards/index.js

```diff
@@ -129,11 +130,13 @@ class Leaderboards extends Component {
 				<div className="woocommerce-dashboard__dashboard-leaderboards">
 					<SectionHeader title={ __( 'Leaderboards', 'wc-admin' ) } menu={ this.renderMenu() } />
 					<div className="woocommerce-dashboard__columns">
-						<div>
-							{ ! hiddenLeaderboardKeys.includes( 'top-products' ) && (
-								<TopSellingProducts query={ this.props.query } totalRows={ rowsPerTable } />
-							) }
-						</div>
+						{ ! hiddenLeaderboardKeys.includes( 'top-products' ) && (
+							<TopSellingProducts query={ this.props.query } totalRows={ rowsPerTable } />
```

Comment:
Minor but now that we are using `this.props.query` multiple times, maybe we should just assign it to a const.